pyproject.toml

```diff
@@ -2,6 +2,60 @@
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "rubrix"
+dynamic = [
+    "keywords",
+    "authors",
+    "maintainers",
+    "license",
+    "requires-python",
+    "version",
+    "description",
+    "readme",
+]
```

Comment:
These are not really dynamic and we could move them from the `setup.cfg` file to the `pyproject.toml` file.

Reply:
I'm not sure if they are been included in the right place. Could you take a look, and do some local tests to verify that I've pushed is fine?